send-message/README.md

```diff
@@ -4,38 +4,15 @@ This action sends a message to [Zulip](https://zulip.com/).
+| Name               | Required? | Details                                                                                                                                                                                                                                                                                            |
+| ------------------ | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| `api-key`          | Yes       | [API key](https://zulip.com/api/api-keys) used to interact with the Zulip API. You can get an API key through Zulip's web interface. You should store the API key [as a secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository). |
+| `content`          | Yes       | The content of the message, formatted using [Zulip Markdown](https://zulip.com/help/format-your-message-using-markdown). Maximum message size of 10000 bytes.                                                                                                                                      |
+| `email`            | Yes       | Email address of the user who owns the API key mentioned above.                                                                                                                                                                                                                                    |
+| `organization-url` | Yes       | Zulip organization canonical URL.                                                                                                                                                                                                                                                                  |     |
+| `to`               | Yes       | For stream messages, either the name or integer ID of the stream. For private messages, either a list containing integer user IDs or a list containing string email addresses.                                                                                                                     |
+| `topic`            | No        | The topic of the message. Only required for stream messages (`type="stream"`), ignored otherwise. Maximum length of 60 characters.                                                                                                                                                                 |
+| `type`             | Yes       | The type of message to be sent. Must be either `private`, for a private message, or `stream`, for a stream message.                                                                                                                                                                                |
```

Comment:
I like the idea putting this in a table, but Github, at least, doesn't seem to be buying it:
![Screenshot 2022-11-29 at 9 25 47 PM](https://user-images.githubusercontent.com/28347/204692555-1d5e0b22-6e0d-4d68-8d2e-c8abda0d3288.png)


https://github.com/zulip/github-actions-zulip/blob/47866b12aa51b83da917aef4acfd4f5c5ef5722c/send-message/README.md

Reply:
Fixed, I had a space between `| --- |` the pipes and dashes in the header separation column, which broke rendering of the entire table. Rich diff now shows the added table correctly, thanks for catching this!